docker/junit5/src/main/java/org/arquillian/cube/docker/junit5/ContainerDsl.java

```diff
@@ -0,0 +1,144 @@
+package org.arquillian.cube.docker.junit5;
+
+import java.io.File;
+import org.arquillian.cube.docker.impl.client.config.Await;
+import org.arquillian.cube.docker.impl.client.containerobject.dsl.BindMode;
+import org.arquillian.cube.docker.impl.client.containerobject.dsl.Container;
+import org.arquillian.cube.docker.impl.client.containerobject.dsl.ContainerBuilder;
+import org.arquillian.cube.docker.junit.rule.ContainerDslRule;
+import org.arquillian.cube.docker.junit.rule.NetworkDslRule;
+import org.arquillian.cube.spi.CubeOutput;
+import org.jboss.shrinkwrap.api.Archive;
+
+public class ContainerDsl {
```

Comment:
How about writing tests for `ContainerDsl`?

Reply:
I thought the same but they are really POJO classes and I don't see much benefit on writing tests for these classes when we have a functional test.

Reply:
If we write some tests around `ContainerDsl`, we'll tests POJO and if any logic around it. Also it'll helpful for end user, if he check the tests around this, end user will get clear idea about how can he use it?

Reply:
Ok I'll write some tests for this case.